### PR TITLE
Initial app

### PR DESCRIPTION
**Pull request by ehaivala**
Simple implementation of Google Maps embed plugin.

Refs RND-37

### CODE REVIEW

**Review by juhakujala — APPROVED** [review]

**Comment by JsseL** [comment]
Soooo... what does this do? :smile:

**Comment by ehaivala** [comment]
This adds CMS plugin that allows Google Maps embeds by address. There's not much else to this really.

**Review by JsseL — APPROVED** [review]

**Comment by ehaivala** [comment]
I changed the readme to ReStructuredText instead of Markdown.

**Comment by JsseL** [comment]
:shipit:

File: .MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+include README.md
```
Comment (suutari-ai) [inline .MANIFEST.in:1]: This is still referencing the README.md.

Comment (ehaivala) [reply .MANIFEST.in:1]: Shoot. I'll fix that. 😅

Comment (ehaivala) [reply .MANIFEST.in:1]: https://github.com/andersinno/cmsplugin-google-maps-ai/pull/2

**Pull request merged**